deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -531,6 +543,50 @@ def __init__(self,
         self._enable_universal_checkpoint()
         self._param_slice_mappings = self._create_param_mapping()
 
+    def get_partition_info_from_bwd_params(self):
+        params_to_parition = {}
```

Comment:
I don't see the return of `params_to_parition`.

Reply:
Sorry for the miss.

Reply:
solved. thanks